ocaml/libs/tracing/tracing.ml

```diff
@@ -832,6 +832,21 @@ module Export = struct
           done
         )
         ()
+
+    let exporter_created = ref None
```

Comment:
I think a better name would be `exporter`. The fact that it exists is expressed in the value.